events/religion_events/religious_interaction_events.txt

```diff
@@ -441,6 +453,12 @@ religious_interaction.1001 = {
 	type = character_event
 	title = {
 		first_valid = {
+			triggered_desc = {
+				trigger = {
+					like_islam_religion_trigger = yes
+				}
+				desc = religious_interaction.1001.t.fatwa
```

Comment:
`religious_interaction.1001.t.fatwa:0 "Fatwa Issued!"`
Don't really care about it, sounds fine to me, but you may want to change it.
Or not.

Reply:
> Don't really care about it, sounds fine to me, but you may want to change it.

Fine to me too.